### ISSUE TITLE
Doesn't scrobble on youtube loop when using another extension to auto-loop

### ISSUE THREAD

**Issue by Californ1a** [web-scrobbler/web-scrobbler#2071]
**Describe the bug**
I know cross-extention compatibility is never really something that tends to work, but I use [this extention](https://chrome.google.com/webstore/detail/looper-for-youtube/iggpfpnahkgpnindfkdncknoldgnccdg) to auto-loop youtube videos, which used to work fine with web scrobbler, but a recent update to the looper added url parameters for the looping functions (`&loop=0` parameter added when enabling looping) which I believe is what causes web scrobbler to give the "This site is not supported for scrobbling" and "Play some music to get started" messages.

**How to reproduce**
1. Install both web scroller and looper
2. Start playing a song on youtube
3. Hit the loop button
4. First one scrobbles but loop doesn't

**Expected behavior**

**Debug logs**
```
inject.js:56 Injecting vendor/jquery.min.js
inject.js:56 Injecting vendor/browser-polyfill.min.js
inject.js:56 Injecting core/content/util.js
inject.js:56 Injecting core/content/reactor.js
inject.js:56 Injecting core/content/filter.js
inject.js:56 Injecting core/content/connector.js
inject.js:56 Injecting connectors/youtube.js
inject.js:56 Injecting core/content/starter.js
extension.js:336 Tab 951049433: Remove controller for YouTube connector
controller.js:541 Tab 951049433: Created controller for YouTube connector
controller.js:541 Tab 951049433: New song detected: {
  "track": "Fear & Delight",
  "artist": "The Correspondents",
  "album": null,
  "albumArtist": null,
  "uniqueID": "ABS-mlep5rY",
  "duration": 200,
  "currentTime": 1,
  "isPlaying": true,
  "trackArt": null
}
pipeline.js:46 Execute processors: 4
scrobble-service.js:90 Send "get info" request: 1
base.js:239 Last.fm: track.getinfo response:
<?xml version="1.0" encoding="UTF-8" ?>
<lfm status="ok"><track><name>Fear &amp; Delight</name>
<mbid>b5732e11-dd2c-4ef2-a711-2d4c07d95379</mbid>
<url>https://www.last.fm/music/The+Correspondents/_/Fear+&amp;+Delight</url>
<duration>193000</duration>
<streamable fulltrack="0">0</streamable>
<listeners>13951</listeners>
<playcount>89215</playcount>
<artist><name>The Correspondents</name>
<mbid>4d8d2c7c-978e-47ac-ba4a-9e798424bf83</mbid>
<url>https://www.last.fm/music/The+Correspondents</url>
</artist>
<album position="2"><artist>The Correspondents</artist>
<title>Puppet Loosely Strung</title>
<mbid>e62507a6-0c0e-412a-bf4a-34809ba3755d</mbid>
<url>https://www.last.fm/music/The+Correspondents/Puppet+Loosely+Strung</url>
<image size="small">https://lastfm.freetls.fastly.net/i/u/34s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="medium">https://lastfm.freetls.fastly.net/i/u/64s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="large">https://lastfm.freetls.fastly.net/i/u/174s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="extralarge">https://lastfm.freetls.fastly.net/i/u/300x300/a90388b70c654086c03a17c1283e44e8.png</image>
</album>
<userplaycount>6</userplaycount><userloved>0</userloved><toptags><tag><name>crime couples</name>
<url>https://www.last.fm/tag/crime+couples</url>
</tag>
</toptags>
</track>
</lfm>

coverartarchive.js:13 Found album artwork via LastFM
controller.js:541 Tab 951049433: Song finished processing: {
  "parsed": {
    "track": "Fear & Delight",
    "artist": "The Correspondents",
    "album": null,
    "albumArtist": null,
    "uniqueID": "ABS-mlep5rY",
    "duration": 200,
    "currentTime": 1,
    "isPlaying": true,
    "trackArt": null
  },
  "processed": {
    "track": "Fear & Delight",
    "album": "Puppet Loosely Strung",
    "artist": "The Correspondents",
    "duration": 193
  },
  "internalId": "ABS-mlep5rY",
  "metadata": {
    "userloved": false,
    "startTimestamp": 1570340514,
    "label": "YouTube",
    "artistThumbUrl": "https://lastfm.freetls.fastly.net/i/u/300x300/a90388b70c654086c03a17c1283e44e8.png",
    "artistUrl": "https://www.last.fm/music/The+Correspondents",
    "trackUrl": "https://www.last.fm/music/The+Correspondents/_/Fear+&+Delight",
    "albumUrl": "https://www.last.fm/music/The+Correspondents/Puppet+Loosely+Strung"
  },
  "flags": {
    "isProcessed": true,
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  }
}
controller.js:541 Tab 951049433: The song will be scrobbled in 100 seconds
controller.js:541 Tab 951049433: The song will be repeated in 200 seconds
scrobble-service.js:106 Send "now playing" request: 1
base.js:239 Last.fm: track.updatenowplaying response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">Fear &amp; Delight</track>
    <artist corrected="0">The Correspondents</artist>
    <album corrected="0">Puppet Loosely Strung</album>
    <albumArtist corrected="0"></albumArtist>
    <ignoredMessage code="0"></ignoredMessage>
  </nowplaying>
</lfm>

controller.js:541 Tab 951049433: Song set as now playing
controller.js:541 Tab 951049433: Received empty state - resetting
controller.js:541 Tab 951049433: New song detected: {
  "track": "Fear & Delight",
  "artist": "The Correspondents",
  "album": null,
  "albumArtist": null,
  "uniqueID": "ABS-mlep5rY",
  "duration": 200,
  "currentTime": 196,
  "isPlaying": true,
  "trackArt": null
}
pipeline.js:46 Execute processors: 4
scrobble-service.js:90 Send "get info" request: 1
base.js:239 Last.fm: track.getinfo response:
<?xml version="1.0" encoding="UTF-8" ?>
<lfm status="ok"><track><name>Fear &amp; Delight</name>
<mbid>b5732e11-dd2c-4ef2-a711-2d4c07d95379</mbid>
<url>https://www.last.fm/music/The+Correspondents/_/Fear+&amp;+Delight</url>
<duration>193000</duration>
<streamable fulltrack="0">0</streamable>
<listeners>13951</listeners>
<playcount>89215</playcount>
<artist><name>The Correspondents</name>
<mbid>4d8d2c7c-978e-47ac-ba4a-9e798424bf83</mbid>
<url>https://www.last.fm/music/The+Correspondents</url>
</artist>
<album position="2"><artist>The Correspondents</artist>
<title>Puppet Loosely Strung</title>
<mbid>e62507a6-0c0e-412a-bf4a-34809ba3755d</mbid>
<url>https://www.last.fm/music/The+Correspondents/Puppet+Loosely+Strung</url>
<image size="small">https://lastfm.freetls.fastly.net/i/u/34s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="medium">https://lastfm.freetls.fastly.net/i/u/64s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="large">https://lastfm.freetls.fastly.net/i/u/174s/a90388b70c654086c03a17c1283e44e8.png</image>
<image size="extralarge">https://lastfm.freetls.fastly.net/i/u/300x300/a90388b70c654086c03a17c1283e44e8.png</image>
</album>
<userplaycount>6</userplaycount><userloved>0</userloved><toptags><tag><name>crime couples</name>
<url>https://www.last.fm/tag/crime+couples</url>
</tag>
</toptags>
</track>
</lfm>

coverartarchive.js:13 Found album artwork via LastFM
controller.js:541 Tab 951049433: Song finished processing: {
  "parsed": {
    "track": "Fear & Delight",
    "artist": "The Correspondents",
    "album": null,
    "albumArtist": null,
    "uniqueID": "ABS-mlep5rY",
    "duration": 200,
    "currentTime": 196,
    "isPlaying": true,
    "trackArt": null
  },
  "processed": {
    "track": "Fear & Delight",
    "album": "Puppet Loosely Strung",
    "artist": "The Correspondents",
    "duration": 193
  },
  "internalId": "ABS-mlep5rY",
  "metadata": {
    "userloved": false,
    "startTimestamp": 1570340566,
    "label": "YouTube",
    "artistThumbUrl": "https://lastfm.freetls.fastly.net/i/u/300x300/a90388b70c654086c03a17c1283e44e8.png",
    "artistUrl": "https://www.last.fm/music/The+Correspondents",
    "trackUrl": "https://www.last.fm/music/The+Correspondents/_/Fear+&+Delight",
    "albumUrl": "https://www.last.fm/music/The+Correspondents/Puppet+Loosely+Strung"
  },
  "flags": {
    "isProcessed": true,
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  }
}
controller.js:541 Tab 951049433: The song will be scrobbled in 100 seconds
controller.js:541 Tab 951049433: The song will be repeated in 200 seconds
scrobble-service.js:106 Send "now playing" request: 1
base.js:239 Last.fm: track.updatenowplaying response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">Fear &amp; Delight</track>
    <artist corrected="0">The Correspondents</artist>
    <album corrected="0">Puppet Loosely Strung</album>
    <albumArtist corrected="0"></albumArtist>
    <ignoredMessage code="0"></ignoredMessage>
  </nowplaying>
</lfm>

controller.js:541 Tab 951049433: Song set as now playing
controller.js:541 Tab 951049433: Received empty state - resetting
```
This only scrobbled once even though the `Song set as now playing` is listed twice as if it was scrobbled twice.

**Environment (please complete the following information)**
-   OS: Windows
-   Browser: Chrome
-   Extension version: 2.22.4

**Additional context**
I've tried most of the other youtube loop extentions on the web store and most all of them don't interact with web scrobbler at all. With nearly every other loop extention, web scroller just continues to say the current song is still playing, not detecting that it has been restarted, whereas looper at least gives some change to web scroller when it loops. I've also already contacted looper devs about getting an option to disable the url parameters, but that was at least 2 or 3 weeks ago and I haven't heard anything back from them, so maybe it could be fixed on web scrobbler's end, since it doesn't work with any of the other auto-looping extensions.

**Comment by alexesprit** [comment]
So, I tried to reproduce the issue, but I have no success. Here are my steps:
1. Install the "Looper for YouTube" extension.
2. Opened [this video][videoUrl] and set up looper to repeat 10 times.

How things were going:
1. The extension recognizes the video correctly.
2. On 50% the track is scrobbled.
3. After the video ends, the looper plays the video again.
4. The extension recongizes the video correctly.
5. On 50% the track is scrobbled.

Then I ended testing.

My debug logs:
<details>
  <summary>Summary (Click to expand)</summary>

  Logs:
```
util.js:163 Tab 102: Created controller for YouTube connector
util.js:163 Tab 102: New song detected: {
  "track": "The Wretched",
  "artist": "Attack Attack!",
  "album": null,
  "albumArtist": null,
  "uniqueID": "jiOepsyiVR8",
  "duration": 242,
  "currentTime": 3,
  "isPlaying": true,
  "trackArt": null
}
pipeline.js:46 Execute processors: 4
scrobble-service.js:90 Send "get info" request: 1
util.js:163 Last.fm: track.getinfo response:
<?xml version="1.0" encoding="UTF-8" ?>
<lfm status="ok"><track><name>The Wretched</name>
<mbid>cacac10b-076b-4edc-a468-74bea210f4cc</mbid>
<url>https://www.last.fm/music/Attack+Attack%21/_/The+Wretched</url>
<duration>246000</duration>
<streamable fulltrack="0">0</streamable>
<listeners>28421</listeners>
<playcount>273626</playcount>
<artist><name>Attack Attack!</name>
<mbid>da876178-b451-4ea6-850f-562c595ff858</mbid>
<url>https://www.last.fm/music/Attack+Attack%21</url>
</artist>
<album position="7"><artist>Attack Attack!</artist>
<title>This Means War</title>
<mbid>15d0aa90-2642-478c-9451-78b80778807c</mbid>
<url>https://www.last.fm/music/Attack+Attack%21/This+Means+War</url>
<image size="small">https://lastfm.freetls.fastly.net/i/u/34s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="medium">https://lastfm.freetls.fastly.net/i/u/64s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="large">https://lastfm.freetls.fastly.net/i/u/174s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="extralarge">https://lastfm.freetls.fastly.net/i/u/300x300/e1fec645d27045d0b5c424fe2e9fd569.png</image>
</album>
<userplaycount>9</userplaycount><userloved>0</userloved><toptags><tag><name>post-hardcore</name>
<url>https://www.last.fm/tag/post-hardcore</url>
</tag>
<tag><name>metalcore</name>
<url>https://www.last.fm/tag/metalcore</url>
</tag>
<tag><name>hardcore</name>
<url>https://www.last.fm/tag/hardcore</url>
</tag>
<tag><name>DJENT</name>
<url>https://www.last.fm/tag/DJENT</url>
</tag>
<tag><name>crabcore</name>
<url>https://www.last.fm/tag/crabcore</url>
</tag>
</toptags>
</track>
</lfm>

coverartarchive.js:13 Found album artwork via LastFM
util.js:163 Tab 102: Song finished processing: {
  "parsed": {
    "track": "The Wretched",
    "artist": "Attack Attack!",
    "album": null,
    "albumArtist": null,
    "uniqueID": "jiOepsyiVR8",
    "duration": 242,
    "currentTime": 3,
    "isPlaying": true,
    "trackArt": null
  },
  "processed": {
    "track": "The Wretched",
    "album": "This Means War",
    "artist": "Attack Attack!",
    "duration": 246
  },
  "internalId": "jiOepsyiVR8",
  "metadata": {
    "userloved": false,
    "startTimestamp": 1571245763,
    "label": "YouTube",
    "artistThumbUrl": "https://lastfm.freetls.fastly.net/i/u/300x300/e1fec645d27045d0b5c424fe2e9fd569.png",
    "artistUrl": "https://www.last.fm/music/Attack+Attack%21",
    "trackUrl": "https://www.last.fm/music/Attack+Attack%21/_/The+Wretched",
    "albumUrl": "https://www.last.fm/music/Attack+Attack%21/This+Means+War",
    "userPlayCount": 9
  },
  "flags": {
    "isProcessed": true,
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  }
}
util.js:163 Tab 102: The song will be scrobbled in 120 seconds
util.js:163 Tab 102: The song will be repeated in 242 seconds
scrobble-service.js:106 Send "now playing" request: 3
util.js:163 Last.fm: track.updatenowplaying response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">The Wretched</track>
    <artist corrected="0">Attack Attack!</artist>
    <album corrected="0">This Means War</album>
    <albumArtist corrected="0"></albumArtist>
    <ignoredMessage code="0"></ignoredMessage>
  </nowplaying>
</lfm>

util.js:163 ListenBrainz: {
  "status": "ok"
}
util.js:163 Libre.fm: track.updatenowplaying response:
<?xml version="1.0"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">The Wretched</track>
    <artist corrected="0">Attack Attack!</artist>
    <album corrected="0">This Means War</album>
    <albumArtist corrected="0"/>
    <ignoredMessage code="0"/>
  </nowplaying>
</lfm>

util.js:163 Tab 102: Song set as now playing
scrobble-service.js:122 Send "scrobble" request: 3
util.js:163 ListenBrainz: {
  "status": "ok"
}
util.js:163 Last.fm: track.scrobble response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <scrobbles ignored="0" accepted="1">
    <scrobble>
      <track corrected="0">The Wretched</track>
      <artist corrected="0">Attack Attack!</artist>
      <album corrected="0">This Means War</album>
      <albumArtist corrected="0"></albumArtist>
      <timestamp>1571245763</timestamp>
      <ignoredMessage code="0"></ignoredMessage>
    </scrobble>
  </scrobbles>
</lfm>

util.js:163 Libre.fm: track.scrobble response:
<?xml version="1.0"?>
<lfm status="ok">
  <scrobbles accepted="1" ignored="0">
    <scrobble>
      <track corrected="0">The Wretched</track>
      <artist corrected="0">Attack Attack!</artist>
      <album corrected="0">This Means War</album>
      <albumArtist corrected="0"/>
      <timestamp>1571245763</timestamp>
      <ignoredMessage code="0"/>
    </scrobble>
  </scrobbles>
</lfm>

util.js:163 Tab 102: Scrobbled successfully
util.js:163 Tab 102: Received empty state - resetting
util.js:163 Tab 102: New song detected: {
  "track": "The Wretched",
  "artist": "Attack Attack!",
  "album": null,
  "albumArtist": null,
  "uniqueID": "jiOepsyiVR8",
  "duration": 242,
  "currentTime": 3,
  "isPlaying": true,
  "trackArt": null
}
pipeline.js:46 Execute processors: 4
scrobble-service.js:90 Send "get info" request: 1
util.js:163 Last.fm: track.getinfo response:
<?xml version="1.0" encoding="UTF-8" ?>
<lfm status="ok"><track><name>The Wretched</name>
<mbid>cacac10b-076b-4edc-a468-74bea210f4cc</mbid>
<url>https://www.last.fm/music/Attack+Attack%21/_/The+Wretched</url>
<duration>246000</duration>
<streamable fulltrack="0">0</streamable>
<listeners>28421</listeners>
<playcount>273626</playcount>
<artist><name>Attack Attack!</name>
<mbid>da876178-b451-4ea6-850f-562c595ff858</mbid>
<url>https://www.last.fm/music/Attack+Attack%21</url>
</artist>
<album position="7"><artist>Attack Attack!</artist>
<title>This Means War</title>
<mbid>15d0aa90-2642-478c-9451-78b80778807c</mbid>
<url>https://www.last.fm/music/Attack+Attack%21/This+Means+War</url>
<image size="small">https://lastfm.freetls.fastly.net/i/u/34s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="medium">https://lastfm.freetls.fastly.net/i/u/64s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="large">https://lastfm.freetls.fastly.net/i/u/174s/e1fec645d27045d0b5c424fe2e9fd569.png</image>
<image size="extralarge">https://lastfm.freetls.fastly.net/i/u/300x300/e1fec645d27045d0b5c424fe2e9fd569.png</image>
</album>
<userplaycount>10</userplaycount><userloved>0</userloved><toptags><tag><name>post-hardcore</name>
<url>https://www.last.fm/tag/post-hardcore</url>
</tag>
<tag><name>metalcore</name>
<url>https://www.last.fm/tag/metalcore</url>
</tag>
<tag><name>hardcore</name>
<url>https://www.last.fm/tag/hardcore</url>
</tag>
<tag><name>DJENT</name>
<url>https://www.last.fm/tag/DJENT</url>
</tag>
<tag><name>crabcore</name>
<url>https://www.last.fm/tag/crabcore</url>
</tag>
</toptags>
</track>
</lfm>

coverartarchive.js:13 Found album artwork via LastFM
util.js:163 Tab 102: Song finished processing: {
  "parsed": {
    "track": "The Wretched",
    "artist": "Attack Attack!",
    "album": null,
    "albumArtist": null,
    "uniqueID": "jiOepsyiVR8",
    "duration": 242,
    "currentTime": 3,
    "isPlaying": true,
    "trackArt": null
  },
  "processed": {
    "track": "The Wretched",
    "album": "This Means War",
    "artist": "Attack Attack!",
    "duration": 246
  },
  "internalId": "jiOepsyiVR8",
  "metadata": {
    "userloved": false,
    "startTimestamp": 1571246004,
    "label": "YouTube",
    "artistThumbUrl": "https://lastfm.freetls.fastly.net/i/u/300x300/e1fec645d27045d0b5c424fe2e9fd569.png",
    "artistUrl": "https://www.last.fm/music/Attack+Attack%21",
    "trackUrl": "https://www.last.fm/music/Attack+Attack%21/_/The+Wretched",
    "albumUrl": "https://www.last.fm/music/Attack+Attack%21/This+Means+War",
    "userPlayCount": 10
  },
  "flags": {
    "isProcessed": true,
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  }
}
util.js:163 Tab 102: The song will be scrobbled in 121 seconds
util.js:163 Tab 102: The song will be repeated in 242 seconds
scrobble-service.js:106 Send "now playing" request: 3
util.js:163 ListenBrainz: {
  "status": "ok"
}
util.js:163 Last.fm: track.updatenowplaying response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">The Wretched</track>
    <artist corrected="0">Attack Attack!</artist>
    <album corrected="0">This Means War</album>
    <albumArtist corrected="0"></albumArtist>
    <ignoredMessage code="0"></ignoredMessage>
  </nowplaying>
</lfm>

util.js:163 Libre.fm: track.updatenowplaying response:
<?xml version="1.0"?>
<lfm status="ok">
  <nowplaying>
    <track corrected="0">The Wretched</track>
    <artist corrected="0">Attack Attack!</artist>
    <album corrected="0">This Means War</album>
    <albumArtist corrected="0"/>
    <ignoredMessage code="0"/>
  </nowplaying>
</lfm>

util.js:163 Tab 102: Song set as now playing
scrobble-service.js:122 Send "scrobble" request: 3
util.js:163 ListenBrainz: {
  "status": "ok"
}
util.js:163 Last.fm: track.scrobble response:
<?xml version="1.0" encoding="UTF-8"?>
<lfm status="ok">
  <scrobbles ignored="0" accepted="1">
    <scrobble>
      <track corrected="0">The Wretched</track>
      <artist corrected="0">Attack Attack!</artist>
      <album corrected="0">This Means War</album>
      <albumArtist corrected="0"></albumArtist>
      <timestamp>1571246004</timestamp>
      <ignoredMessage code="0"></ignoredMessage>
    </scrobble>
  </scrobbles>
</lfm>

util.js:163 Libre.fm: track.scrobble response:
<?xml version="1.0"?>
<lfm status="ok">
  <scrobbles accepted="1" ignored="0">
    <scrobble>
      <track corrected="0">The Wretched</track>
      <artist corrected="0">Attack Attack!</artist>
      <album corrected="0">This Means War</album>
      <albumArtist corrected="0"/>
      <timestamp>1571246004</timestamp>
      <ignoredMessage code="0"/>
    </scrobble>
  </scrobbles>
</lfm>

util.js:163 Tab 102: Scrobbled successfully
```
Content
</details>

The interesting thing in your logs is you have two "now playing" requests, but one "scrobble" request only. If you take a look at my logs, you can notice two "now playing" and two "scrobble" requests.

Did you stop the video before 50% (the half of duration when the extension sends "scrobble" request to Last.fm)? If no, could you record a screencast where you reproduce the issue?

[videoUrl]: https://www.youtube.com/watch?v=jiOepsyiVR8

**Comment by Californ1a** [comment]
>The interesting thing in your logs is you have two "now playing" requests, but one "scrobble" request only. If you take a look at my logs, you can notice two "now playing" and two "scrobble" requests.

>Did you stop the video before 50% (the half of duration when the extension sends "scrobble" request to Last.fm)? If no, could you record a screencast where you reproduce the issue?

I actually played the video many more than 2 times, it only shows up twice in the log because web scrobbler says it doesn't recognize the website after the 1st play and stops putting anything more in the log unless I refresh the page. It detects that the video was looped, because it lists the song as playing again, but it says the website isn't supported after that and doesn't scrobble the 2nd time, then doesn't recognize any further loops.

If I manually hit replay after the video finishes, it scrobbles again as many times as I hit replay, but if I use *any looping extension I can find on the web store*, then it either scrobbles once and just says the song is still playing on the 2nd play without scrobbling again, or it scrobbles once and then says the site isn't supported on the 2nd play. I am still using the old YouTube layout, via [a userscript](https://greasyfork.org/en/scripts/32906-get-me-old-youtube) to change the cookie for it, if that makes any difference.

**Comment by alexesprit** [comment]
> I am still using the old YouTube layout, via a userscript to change the cookie for it, if that makes any difference.

Could you try to reproduce the issue w/o that userscript?

> The interesting thing in your logs is you have two "now playing" requests, but one "scrobble" request only. If you take a look at my logs, you can notice two "now playing" and two "scrobble" requests.

I was wrongs when I said this. There's no "scrobble" requests in your logs, but two "now playing" requests.

**Comment by Californ1a** [comment]
>Could you try to reproduce the issue w/o that userscript?

Yes, YouTube still serves my account the old layout for now even with the userscript disabled and cookies cleared because I've forced it for so long, they gave up being aggressive with the layout change. Forcing the new layout does make loops scrobble *somewhat* - I have to mouseover the video to get the controls to open for it to recognize the site. Without doing that, it says site not supported.

**Comment by alexesprit** [comment]
Closed in favor of #2194.
